src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AotTests.cs

```diff
@@ -168,6 +182,7 @@ public void BuildBasicApplicationReleaseProfiledAotWithoutDefaultProfile ()
 			proj.SetProperty (KnownProperties.TargetFrameworkVersion, "v5.1");
 			proj.SetAndroidSupportedAbis (supportedAbis);
 			proj.SetProperty ("EnableLLVM", enableLLVM.ToString ());
+			proj.SetProperty ("AndroidUseAssembliesBlob", usesAssemblyBlobs.ToString ());
```

Comment:
I think we want a "better" name than "blob".  I think I previously suggested "AssemblyContainer", though perhaps "Archive" could be a shorter replacement?